### PR TITLE
Documentation improvement

### PR DESCRIPTION
**Pull request by seregazhuk**
This PR is the same as #297 
Removes word `function` from middleware description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -860,7 +860,7 @@ $server = new Server(array(
 
 Similarly, you can use the result of the `$next` middleware request handler
 function to modify the outgoing response.
-Note that as per the above documentation, the `$next` function may return a
+Note that as per the above documentation, the `$next` may return a
```
Comment (jsor) [inline README.md]: "...the `$next` may return a..." reads a bit odd imho.

Comment (seregazhuk) [reply README.md]: May be "the `$next` variable" or "the callable `$next`"?

Comment (jsor) [reply README.md]: In the other paragraph ↓, its called "`$next` middleware request handler", maybe that?

Comment (seregazhuk) [reply README.md]: Fixed.

**Pull request merged**